contentcuration/contentcuration/static/js/edit_channel/share/views.js

```diff
@@ -45,12 +46,18 @@ var ShareView = BaseViews.BaseView.extend({
     },
 
     render: function() {
+        var share_modes = this.get_share_modes();
         this.$el.html(this.template({
             channel:this.model.toJSON(),
             user: this.current_user.toJSON(),
-            share_modes: this.get_share_modes()
+            share_modes: share_modes,
```

Comment:
This is so that it doesn't call it twice when getting the length and the actual modes available